tools/spectral/ipa/rulesets/IPA-112.yaml

```diff
@@ -63,8 +65,9 @@ rules:
     message: '{{error}} https://mdb.link/mongodb-atlas-openapi-validation#xgen-IPA-112-boolean-field-names-avoid-is-prefix'
     severity: warn
     given:
-      - '$.components.schemas..properties[*]~'
-      - '$.paths..requestBody.content[?(@property.match(/json$/i))].schema..properties[*]~'
-      - '$.paths..responses..content[?(@property.match(/json$/i))].schema..properties[*]~'
+      - '$.components.schemas..properties'
+      - '$.paths..requestBody.content[?(@property.match(/json$/i))].schema..properties'
```

Comment:
[nit] JSON filtering can be done as option for rule configuration and checked in code.